Sandbox/Sandbox/AppDelegate.swift

```diff
@@ -7,75 +7,107 @@ import IdentitySdkGoogle
-        ),
-        providersCreators: [
-            FacebookProvider(),
-            GoogleProvider(),
-            WebViewProvider()
-        ],
-        storage: UserDefaultsStorage()
+
+    public static let storage = SecureStorage()
+    public static let authKey = "AUTH_TOKEN"
+
+    let reachfive: ReachFive = ReachFive(
+            sdkConfig: SdkConfig(
+                    domain: "sdk-mobile-sandbox.reach5.net",
+                    clientId: "TYAIHFRJ2a1FGJ1T8pKD",
+                    scheme: "reachfive-TYAIHFRJ2a1FGJ1T8pKD://callback"
+            ),
+            providersCreators: [
+                FacebookProvider(),
+                GoogleProvider(),
+                WebViewProvider()
+            ],
+            storage: UserDefaultsStorage()
     )
-    
+
     static func reachfive() -> ReachFive {
         let app = UIApplication.shared.delegate as! AppDelegate
         return app.reachfive
     }
-    
+
     static func shared() -> AppDelegate {
         return UIApplication.shared.delegate as! AppDelegate
     }
-    
+
     static func createAlert(title: String, message: String) -> UIAlertController {
         let alert = UIAlertController(
-            title: title,
-            message: message,
-            preferredStyle: UIAlertController.Style.alert
+                title: title,
+                message: message,
+                preferredStyle: UIAlertController.Style.alert
         )
         alert.addAction(UIAlertAction(title: "OK", style: UIAlertAction.Style.default, handler: nil))
         return alert
     }
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        print("application:didFinishLaunchingWithOptions")
         reachfive.addPasswordlessCallback { result in
             print("addPasswordlessCallback \(result)")
         }
-        
+
         return true
     }
-    
-    func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
+
+    func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey: Any] = [:]) -> Bool {
         return reachfive.application(app, open: url, options: options)
     }
-    
+
     func applicationWillResignActive(_ application: UIApplication) {
+        print("applicationWillResignActive")
         // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
         // Use this method to pause ongoing tasks, disable timers, and invalidate graphics rendering callbacks. Games should use this method to pause the game.
     }
 
     func applicationDidEnterBackground(_ application: UIApplication) {
+        print("applicationDidEnterBackground")
```

Comment:
Do we still want the print statements for the sandbox?

Reply:
This sandbox is for education purpose for now. And we still haven't reactivated the Alamofire logging.

Reply:
So not ideal, but it is helpful for now.